### PR TITLE
removing kubectl gcloud component

### PR DESCRIPTION
**Pull request by RagniGarg97**
We are getting this error while building all-in-one image on ppc architecture:

```
 > [stage-1  9/10] RUN wget -q https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk.tar.gz &&     tar xzf google-cloud-sdk.tar.gz -C / &&     rm google-cloud-sdk.tar.gz &&     gcloud components install alpha beta kubectl gsutil &&     gcloud info | tee /workspace/gcloud-info.txt:
------
Error: failed to solve: rpc error: code = Unknown desc = executor failed running [/bin/sh -c wget -q https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk.tar.gz &&     tar xzf google-cloud-sdk.tar.gz -C / &&     rm google-cloud-sdk.tar.gz &&     gcloud components install alpha beta kubectl gsutil &&     gcloud info | tee /workspace/gcloud-info.txt]: exit code: 1
{"component":"unset","error":"wrapped process failed: exit status 1","file":"/go/test-infra/prow/entrypoint/run.go:80","func":"k8s.io/test-infra/prow/entrypoint.Options.Run","level":"error","msg":"Error executing test process","severity":"error","time":"2021-03-31T11:14:52Z"}
```
This error is coming up  for kubectl component only., alpha,beta and gsutil are getting installed but not  kubectl via gcloud (for ppc).

As we are using the below commands to install kubectl, we can remove kubectl from gcloud component to avoid this error.
```
RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/`go env GOARCH`/kubectl && \
chmod +x ./kubectl && \
mv ./kubectl /usr/local/bin/kubectl
```

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/171#" title="Author self-approved">RagniGarg97</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mkumatag** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @mkumatag` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by Rajalakshmi-Girish** [comment]
@ragni Can you please update the description with the reason for removal with proper error messages?
It will be useful for tracking purposes.

**Comment by RagniGarg97** [comment]
> @ragni Can you please update the description with the reason for removal with proper error messages?
> It will be useful for tracking purposes.

Done

**Review by mkumatag — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/171#pullrequestreview-634112098" title="LGTM">mkumatag</a>*, *<a href="https://github.com/ppc64le-cloud/test-infra/pull/171#" title="Author self-approved">RagniGarg97</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
